cypress/component/app/pageHome.cy.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
```

Comment:
what is this file supposed to be?

Reply:
It's a sample component test for cypress to check the ci but I will be removing it and split the CI part into another PR